docs/Embedded/EnkluVolume.md

```diff
@@ -0,0 +1,18 @@
+---
+id: EnkluVolume
+title: Embedding an Experience
+---
+
+_\[Note: Access to this feature requires a [Studio Subscription](https://enklu.com/pricing). [Contact us](/contact) for details.\]_
+
+To embed a scene in your Unity project, add an EnkluVolume prefab to your scene. Find it by searching for "EnkluVolume" in the project tab and ensuring the "In Packages" option is selected. Once the prefab is in your scene, select it and log in to your [Enklu Cloud account](https://cloud.enklu.com) in the inspector tab.
+
+![Enklu Volume](/img/product/embedded/VolumeImport.gif)
+
+Once you logged in, use the inspector tab to set an existing experience to embed or create a new one. You can embed an entire Enklu experience in your scene or just part of one by selecting or creating an element.
```

Comment:
"Once you logged in" - I think this is grammatically correct, but I keep reading it funny. Maybe something like "After you've logged in..."?

Reply:
I think it was _meant_ to be "Once you've logged in" but "After" feels a little friendlier anyway.